layout_catalog/src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,109 @@
+/* #region page__catalog */
+.page__catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  box-sizing: border-box;
+
+  width: $catalog-width;
+  margin: 0 auto;
```

Comment:
move it to page file

Reply:
to create a new page.scss or include it in .page in the existing reset.scss?